Sources/IdentifiedCollections/Identified/Identified.swift

```diff
@@ -58,3 +58,7 @@ extension Identified: Encodable where ID: Encodable, Value: Encodable {}
 extension Identified: Equatable where Value: Equatable {}
 
 extension Identified: Hashable where Value: Hashable {}
+
+#if canImport(_Concurrency) && compiler(>=5.5.2)
+extension Identified: Sendable where ID: Sendable, Value: Sendable {}
+#endif
```

Comment:
We target 5.6+ these days, so don't think this is needed!

```suggestion
extension Identified: Sendable where ID: Sendable, Value: Sendable {}
```